colors/build.js

```diff
@@ -2,29 +2,50 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
+function createColor(color, element, outputType) {
+  var output = [];
+  let prefix;
+  let lastColor;
+
+  switch(outputType) {
+    case 'scss':
+      prefix = '$';
+      break
+    case 'css':
+    default:
+      prefix = '  --';
+  }
+
   for (var variant in element) {
     if (element.hasOwnProperty(variant)) {
       var value = element[variant];
-      rv.push(`  --${color}-${variant}: ${value};\n`);
+      output.push(`${prefix}${color}-${variant}: ${value};\n`);
     }
   }
-  return rv;
+  return output;
 }
 
-let rv = [`/* Firefox Colors CSS Variables v${metadata.version} */
+
+let cssOutput = [`/* Firefox Colors CSS Variables v${metadata.version} */
 
 :root {
 `];
+
+let sassOutput = [`/* Firefox Colors SCSS Variables v${metadata.version} */
+
+`];
+
 for (var color in colors) {
   if (colors.hasOwnProperty(color)) {
     var element = colors[color];
-    rv.push(...createColor(color, element));
+    cssOutput.push(...createColor(color, element));
```

Comment:
The switch was only for assuming there might be additional future cases depending on the needs of other web properties who like us might prefer to use pre-processor vars over CSS vars. Can remove if it's better to wait to add that only when needed.